app/api/src/tests/api/api_v1/list_todos.sh

```diff
@@ -0,0 +1,7 @@
+for filename in *.py; do
+    if grep -q '# TODO:' $filename;
+    then
+        echo "### $filename"
+        sed -n -e '/^# TODO:/p' $filename| sed -e "s/^# TODO://" | sed 's/.*/- [ ]&/'
+    fi
+done
```

Comment:
We might not need this. We can use this plugin: 
https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

Reply:
This is not highlighter. It lists the TODOs for github issue. The code highlighters already highlight TODOs.

Reply:
I get it. So you are exporting to copy it when creating the epic issues (markdown). The highlight plugin also can export but not sure if markdown.
Anyways, maybe we can extend this also for the front end, as now this works only for the python files.

Reply:
This PR is for separating env variables of docker-compose and uvicorn Ports. In related issues there are more details.